Makefile

```diff
@@ -28,6 +28,8 @@ help:
 	@echo "  reindex           to reindex everything in elasticsearch, for AMO"
 	@echo "  flake8            to run the flake8 linter"
 	@echo "Check the Makefile  to know exactly what each target is doing. If you see a "
+	@echo "target using something like $(SETTINGS), you can make it use another value:"
+	@echo "  make SETTINGS=settings_mine docs"
```

Comment:
That should be `$$(SETTINGS)`. And there's an extra space before the 'to' on the above line. I'm not really sure I like any of these three sentences, to be honest...